docs/src/content/docs/guides/i18n.mdx

```diff
@@ -37,6 +37,11 @@ Starlight provides built-in support for multilingual sites, including routing, f
    					label: 'العربية',
    					dir: 'rtl',
    				},
+   				// Korean docs in `src/content/docs/ko/`
+   				ko: {
+   					label: '한국어',
+   					lang: 'ko-KR',
+   				},
```

Comment:
This code block is more to give an example of a few languages and doesn’t need to show every possible language, so I think we can remove Korean here:
```suggestion
```